bim2sim/elements/mapping/attribute.py

```diff
@@ -5,7 +5,7 @@
 import pint
 
 from bim2sim.kernel.decision import RealDecision, Decision, \
-    DecisionBunch
+    DecisionBunch, BoolDecision, StringDecision
```

Comment:
are BoolDecision, StringDecision new Objects in bim2sim.kernel.decision?
If yes, than this change is missing in this pr, or not?

Reply:
Nope, those decision classes already exist.

Reply:
There is also ListDecision and Guide vision, but I don't see those as relevant for an attribute.